Description: link mapnik against dl
Author: Jérémy Lal <kapouer@melix.org>
Forwarded: not yet
Last-Update: 2013-08-29
--- a/src/build.py
+++ b/src/build.py
@@ -57,7 +57,7 @@
 system = 'boost_system%s' % env['BOOST_APPEND']
 
 # clear out and re-set libs for this env
-lib_env['LIBS'] = ['freetype','z',env['ICU_LIB_NAME'],filesystem,system,regex]
+lib_env['LIBS'] = ['freetype','z',env['ICU_LIB_NAME'],filesystem,system,regex,'dl']
 
 if env['PROJ']:
    lib_env['LIBS'].append('proj')
