Description: fix raster calls wrong number of arguments
Forwarded: TODO
Last-Update: 2014-10-24
Author: Jérémy Lal <kapouer@melix.org>
--- a/plugins/input/rasterlite/rasterlite_featureset.cpp
+++ b/plugins/input/rasterlite/rasterlite_featureset.cpp
@@ -114,7 +114,7 @@
         {
             if (size > 0)
             {
-                mapnik::raster_ptr rasterp = std::make_shared<mapnik::raster>(intersect, width, height);
+                mapnik::raster_ptr rasterp = std::make_shared<mapnik::raster>(intersect, width, height, 1.0);
                 mapnik::image_data_32 & image = rasterp->data_;
                 image.set(0xffffffff);
 
