Description: Fix 'intialize' typo, replace with 'initialize'.
Author: Bas Couwenberg <sebastic@debian.org>
Forwarded: https://github.com/mapnik/mapnik/pull/3480

--- a/src/map.cpp
+++ b/src/map.cpp
@@ -695,7 +695,7 @@ featureset_ptr Map::query_point(unsigned
 {
     if (!current_extent_.valid())
     {
-        throw std::runtime_error("query_point: map extent is not intialized, you need to set a valid extent before querying");
+        throw std::runtime_error("query_point: map extent is not initialized, you need to set a valid extent before querying");
     }
     if (!current_extent_.intersects(x,y))
     {
--- a/src/marker_cache.cpp
+++ b/src/marker_cache.cpp
@@ -267,7 +267,7 @@ std::shared_ptr<mapnik::marker const> ma
                 }
                 else
                 {
-                    MAPNIK_LOG_ERROR(marker_cache) << "could not intialize reader for: '" << uri << "'";
+                    MAPNIK_LOG_ERROR(marker_cache) << "could not initialize reader for: '" << uri << "'";
                     return std::make_shared<mapnik::marker const>(mapnik::marker_null());
                 }
             }
