Description: Fix 'occured' typo, replace with 'occurred'.
Author: Bas Couwenberg <sebastic@debian.org>
Forwarded: https://github.com/mapnik/mapnik/pull/3480

--- a/src/load_map.cpp
+++ b/src/load_map.cpp
@@ -755,7 +755,7 @@ void map_parser::parse_layer(Map & map,
                 }
                 catch (...)
                 {
-                    throw config_error("Unknown exception occured attempting to create datasoure for layer '" + lyr.name() + "'");
+                    throw config_error("Unknown exception occurred attempting to create datasoure for layer '" + lyr.name() + "'");
                 }
             }
         }
